docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,138 @@
+## Getting started
+
+### Introduction
+
+[CARTO VL](https://github.com/cartodb/carto-vl) is a JavaScript library to create custom location intelligence applications that leverage the power of [CARTO](https://carto.com/). It uses [WebGL](https://www.khronos.org/webgl/) to enable powerful vector maps.
+
+It relies on [Mapbox GL](https://www.mapbox.com/mapbox-gl-js/api) to render the basemaps and it can be used for other functionality too, read [Mapbox GL documentation](https://www.mapbox.com/mapbox-gl-js/api/) for more information. However, keep in mind that CARTO VL layers can only be controlled with the CARTO VL API (this documentation) and that Mapbox GL native layers can only be controlled with the Mapbox GL API. Therefore, CARTO VL expressions cannot be used for Mapbox GL layers and vice versa.
+
```

Comment:
I miss an **introduction to the guide** itself, not the library. Something like, "In this guide, you will learn how to create a basic interactive map, displaying a dataset from CARTO on the top of a world basemap".  I think we should add this kind of intro to every guide.

Maybe just in this case, cause this is probably the entry point for many people, but I miss some kind of **expectations on the user**, like: "There are no previous requirements to complete this guide, but a basic understanding on HTML and JavaScript is helpful. You will need just a simple text editor and an Internet connection".

Reply:
YES! I have not thought about it but I think it is really useful and it is actually what you expect in an introduction ✨